presto-tpcds/src/test/java/com/facebook/presto/tpcds/statistics/TestTpcdsLocalStats.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tpcds.statistics;
+
+import com.facebook.presto.Session;
+import com.facebook.presto.testing.LocalQueryRunner;
+import com.facebook.presto.tests.statistics.StatisticsAssertion;
+import com.facebook.presto.tpcds.TpcdsConnectorFactory;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.SystemSessionProperties.USE_NEW_STATS_CALCULATOR;
+import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
+import static com.facebook.presto.tests.statistics.MetricComparisonStrategies.absoluteError;
+import static com.facebook.presto.tests.statistics.MetricComparisonStrategies.defaultTolerance;
+import static com.facebook.presto.tests.statistics.MetricComparisonStrategies.noError;
+import static com.facebook.presto.tests.statistics.Metrics.OUTPUT_ROW_COUNT;
+import static java.util.Collections.emptyMap;
+
+public class TestTpcdsLocalStats
+{
+    private final StatisticsAssertion statisticsAssertion;
+
+    public TestTpcdsLocalStats()
+            throws Exception
+    {
+        Session defaultSession = testSessionBuilder()
+                .setCatalog("tpcds")
+                .setSchema("sf1") // TODO decision to make (slower vs less practical). Also "tiny" doesn't work now
+//                .setSystemProperty(PUSH_PARTIAL_AGGREGATION_THROUGH_JOIN, "true") TODO why is this set in tpch's test??
```

Comment:
i assume you've read the comment and concluded this system property ain't necessary. Shall i remove it from tpch local stats test too?